code/modules/clothing/head/helmet.dm

```diff
@@ -261,18 +265,30 @@
 		camera.c_tag = "Unknown"
 	..()
 
-
 /obj/item/clothing/head/helmet/marine/proc/add_hugger_damage() //This is called in XenoFacehuggers.dm to first add the overlay and set the var.
 	if(flags_marine_helmet & HELMET_DAMAGE_OVERLAY && !(flags_marine_helmet & HELMET_IS_DAMAGED))
 		helmet_overlays["damage"] = image('icons/obj/clothing/cm_hats.dmi',icon_state = "hugger_damage")
 		flags_marine_helmet |= HELMET_IS_DAMAGED
 		update_icon()
 		desc += "\n<b>This helmet seems to be scratched up and damaged, particularly around the face area...</b>"
 
+//Macros used to speed up making helmets for vox heads
+#define EXCLUDE_VOX species_restricted = list("exclude", "Vox")
```

Comment:
should probably rename this define to nonhuman if we want to add more species in future

Reply:
Will do in a second.

Reply:
Done.